Gosu/Window.hpp

```diff
@@ -50,11 +50,21 @@ namespace Gosu
         //! function.
         Window(unsigned width, unsigned height, bool fullscreen = false,
             double update_interval = 16.666666);
+        Window(unsigned width, unsigned height, bool fullscreen = false, bool resizable = false,
+            double update_interval = 16.666666);
         virtual ~Window();
 
         unsigned width() const;
         unsigned height() const;
         bool fullscreen() const;
+        bool resizable() const;
+
+        //! Called when the size of the window has changed.
+        virtual void size_changed() {};
+
+        //! Sets the minimum size the window can be.
+        void minimum_size(unsigned width, unsigned height);
```

Comment:
Don't need this as well. This could be done in resize as well as checks for maximum size, ratio and adjustments to your drawing code to the new size.